src/web/blocks.rs

```diff
@@ -52,41 +54,49 @@ pub struct BlockDTO {
     height: u32,
     timestamp: Timestamp,
     block_hash: SerScaleHex<Hash>,
-    parent_block_hash: SerScaleHex<Hash>,
-    transactions_merkle_root_hash: SerScaleHex<Hash>,
-    rejected_transactions_merkle_root_hash: SerScaleHex<Hash>,
-    invalidated_blocks_hashes: Vec<SerScaleHex<Hash>>,
-    transactions: Vec<SerScaleHex<VersionedValidTransaction>>,
-    rejected_transactions: Vec<SerScaleHex<VersionedRejectedTransaction>>,
-    view_change_proofs: Vec<SerScaleHex<Hash>>,
+    parent_block_hash: Option<HashOf<VersionedCommittedBlock>>,
+    transactions_merkle_root_hash: Option<HashOf<MerkleTree<VersionedSignedTransaction>>>,
+    rejected_transactions_merkle_root_hash: Option<HashOf<MerkleTree<VersionedSignedTransaction>>>,
+    // invalidated_blocks_hashes: Vec<SerScaleHex<Hash>>,
+    transactions: Vec<TransactionValue>,
+    // rejected_transactions: Vec<SerScaleHex<VersionedSignedTransaction>>,
+    // view_change_proofs: Vec<SerScaleHex<Hash>>,
```

Comment:
1. Why do you delete `SerScaleHex` wrap?
2. Why do you comment out `invalidated_blocks_hashes`, `rejected_transactions` and `view_change_proofs`?

Reply:
1. During the initial refactoring process, I deleted the ```SerScaleHex``` wrap because it was causing some ```type/conversion``` errors. I recognize that using ```SerScaleHex``` is a more appropriate approach. As a result, I made the change and updated the appropriate fields with the latest commit. 

2. I decided to comment out ```invalidated_blocks_hashesa and rejected_transactions``` because, during my initial research, it wasn't clear what was happening with ```rejected_transactions```. As for ```invalidated_blocks_hashes```, they seemed closely related to the notion of rejection by consensus, but since ```rejected_transactions``` were interleaved in the ```iroha2-dev``` branch, I chose to ignore them. Additionally, there was no information about ```view_change_proof``` in the documentation and the repository used to open the issue indicated it was **dead code.** 
With the latest commit, ```rejected_transactions and invalidated_block_hashes``` are treated as ```0``` or ```empty vector``` to handle them.